experimental/torch_xla2/torch_xla2/ops/ops_registry.py

```diff
@@ -0,0 +1,57 @@
+import torch
+import jax
+import jax.numpy as jnp
+
+import dataclasses
+from typing import TypeAlias, Callable, ParamSpec, Any, Union, Dict
+
+P = ParamSpec('P')
+
+TorchValue: TypeAlias = Union[torch.Tensor, torch.dtype, 'TorchCallable', Any]
```

Comment:
It looks like these type aliases are repeated from `interop.py`. Do you think it would be useful to have a shared `types` module to hold these in one place?